### DISCUSSION TITLE
How do you the "ctrl+click" thing that's supposed to automagically link the node inputs?

### DISCUSSION THREAD

**Discussion by stephantual** [rgthree/rgthree-comfy#84] [General]
First, since it's the first dicussion in  repo, A HUGE THANK YOU to RGthree for this amazing set of nodes, which I'll eventually figure out how to use correctly. I'll keep my question to the ones reddit couldn't answer:

Question 1 - In the docs it says " _When dragging a Context output over a nother node, hold down "ctrl" and release to automatically connect the other Context outputs to the hovered node.". Okay but how ? I tried clicking various part of the node and releasing on various part of another context node or even a Ksampler (OG comfy version), no dice.

No biggy and it's probably user errror but I'm curious to know how that works :)
Thank you!

**Comment by rgthree** [comment]
>  A HUGE THANK YOU to RGthree for this amazing set of nodes

Thanks :)

For the auto-connect from a Context Node:

1. Click and hold the Context Node's CONTEXT output to start dragging the noodle connection
2. Pull it over another node's inputs
3. Release the click, and the Context Node should try its best to connect all valid empty inputs to itself (it may miss some, so you'll want to double check).

To _force_ all inputs:

1. Click and hold the Context Node's CONTEXT output to start dragging the noodle connection
2. Pull it over another node's inputs
3. Press and hold the CTRL key
4. Release the click, and the Context Node should try its best to connect all valid inputs to itself, disconnecting and overriding any inputs it wants to connect (it may miss some, so you'll want to double check).
5. (You can also release the CTRL key now too).



But, a video may be able to show it better, notice when the "CTRL" flashes on screen signaling when the key is being pressed:

https://github.com/rgthree/rgthree-comfy/assets/476360/0e3b5823-6321-4f6b-bdc5-e84041988ea1

**Reply by stephantual** [reply]
Oh!!!! of course! Thank you ever so much 👍 This is amazing and makes life a lot simpler. At the risk of souding like a stalker, I saw on reddit you were thinking of implementing dynamic context (a bit like the new Junction nodes in 0246) - are you still thinking about those? 

Cheers and again thank you! 💯

**Comment by RoyDingZF** [comment]
I can not replicate this for some reason, have to do it one by one

**Reply by stephantual** [reply]
Try to click from the 'dot' of the context to the nodes. Just tested it on ### ComfyUI Revision: 2096 [6c6a3925] | Released on '2024-04-02',  with latest RG3, it works .
